packages/state-importer/src/main.js

```diff
@@ -96,34 +96,36 @@ export default class StateImport {
     this._resetSummary()
   }
 
-  static _buildPredicate(states: Array<StateData>) {
-    return `key in ("${states.map(state => state.key).join('", "')}")`
+  static _buildPredicate(states: Array<StateData>): string {
+    return `key in ("${states
+      .map((state: StateData): string => state.key)
+      .join('", "')}")`
   }
 
   // Wrapper function for compatibility with CLI
-  processStream(chunk: Array<StateData>, cb: () => mixed) {
+  processStream(chunk: Array<StateData>, cb: () => mixed): Promise<mixed> {
     this.logger.debug(`Starting conversion of ${chunk.length} states`)
     return this._processBatches(chunk).then(cb)
     // No catch block as errors will be caught in the CLI
   }
 
   // Public function for direct module usage
-  run(states: Array<StateData>) {
+  run(states: Array<StateData>): StateData {
     return this._processBatches(states)
   }
 
-  _processBatches(states: Array<StateData>) {
+  _processBatches(states: Array<StateData>): Object {
     const predicate = StateImport._buildPredicate(states)
     const service = this._createService()
     const uri = service.where(predicate).build()
     const existingStatesRequest = StateImport._buildRequest(uri, 'GET')
     return this.client
       .execute(existingStatesRequest)
-      .then(({ body: { results: existingStates } }: Object) =>
+      .then(({ body: { results: existingStates } }: Object): Object =>
         this._createOrUpdate(states, existingStates)
       )
-      .then(() => Promise.resolve())
-      .catch(error => {
+      .then((): Promise<any> => Promise.resolve())
```

Comment:
This one cant be `any` when it literally resolves to undefined next to it.
`Promise.resolve()`